performance_test/src/experiment_configuration/experiment_configuration.cpp

```diff
@@ -125,7 +125,7 @@ void ExperimentConfiguration::setup(int argc, char ** argv)
 #endif
   ;
   po::variables_map vm;
-  po::store(parse_command_line(argc, argv, desc), vm);
+  po::store(po::command_line_parser(argc, argv).options(desc).allow_unregistered().run(), vm);
```

Comment:
@BMarchi What is this change doing?

Reply:
Eloquent's [ros2 launch](https://github.com/ros2/launch_ros/blob/e64cf54116f365e171a2a06c1dbe65815eb76550/launch_ros/launch_ros/actions/node.py#L136) introduces a --ros-args argument by default.